packages/page-practice/lib/practice/Presenter.tsx

```diff
@@ -166,10 +166,21 @@ export class Presenter extends PureComponent<Props, State> {
   };
 
   private handleKeyDown = (ev: KeyEvent): void => {
+    const modifierKeys = ["CapsLock", "NumLock"];
```

Comment:
This line should be moved to where we create a `KeyEvent` instance from a `KeyboardEvent`.